app/src/main/java/org/fossasia/openevent/general/order/OrderCompletedFragment.kt

```diff
@@ -109,6 +118,26 @@ class OrderCompletedFragment : Fragment() {
         return rootView
     }
 
+    private fun displayRateEventyayAlertDialog() {
+        if (!preferences.getBoolean(DISPLAY_RATING_DIALOG, true))
+            return
+        val layout = layoutInflater.inflate(R.layout.dialog_rate_us, null)
+        val alertDialog = AlertDialog.Builder(requireContext())
+            .setView(layout)
+            .setNeutralButton(getString(R.string.no_thanks)) { _, _ ->
+                preferences.putBoolean(DISPLAY_RATING_DIALOG, false)
+            }.setPositiveButton(getString(R.string.maybe_later)) { _, _ ->
+                preferences.putBoolean(DISPLAY_RATING_DIALOG, true)
+            }.show()
+        layout.rateEventyayButton.setOnClickListener {
+            alertDialog.dismiss()
+            val appPackageName = "com.eventyay.attendee"
```

Comment:
Don't hardcode. Fetch actual

Reply:
@iamareebjamal But there is something wrong. The package name is different. In the repo, it is `org.fossasia.openevent.general`
https://github.com/fossasia/open-event-android/blob/91eda72bab0ed86f564ed26d259b7acb1eff7c7a/app/src/main/AndroidManifest.xml#L3 And on play store it is `com.eventyay.attendee` https://play.google.com/store/apps/details?id=com.eventyay.attendee

Reply:
See build.gradle

Reply:
Updated